src/Exchange_Rates.json

```diff
@@ -0,0 +1,23 @@
+{
+  "success": true,
+  "base": "USD",
+  "date": "2023-07-17",
+  "rates": {

```

Comment:
Are these all the currencies that we support in AME? If yes, then, we don't need a USD fallback.

Reply:
Removed USD fallback